pipelines/vars/forklift_katello.yml

```diff
@@ -25,6 +25,6 @@ proxy_box:
       foreman_installer_disable_system_checks: true
       foreman_installer_scenario: foreman-proxy-content
       foreman_installer_additional_packages:
-        - foreman-proxy-content
+        - foreman-installer-foreman-proxy-content
```

Comment:
this package doesn't exist?

Reply:
Yea, today we have foreman-installer-katello that covers both.

Reply:
Right, I was getting ahead of myself. This is something I would like to have at some point.